test_service.py

```diff
@@ -0,0 +1,58 @@
+import grpc
+import pytest
+import service
+from hwsc_file_transaction_svc_pb2 import FileTransactionRequest
+
+
+# TODO refactor
```

Comment:
I will let you guys figure out where to put this Fake or Mock classes. We will definitely add more in the future depending on the `context` used by GRPC.